.github/workflows/docker-build.yml

```diff
@@ -0,0 +1,46 @@
+name: Docker
+
+on:
+  workflow_call:
+
+concurrency:
+  group: ${ {github.event_name }}-${{ github.workflow }}-${{ github.ref }}-docker-build
+  cancel-in-progress: ${{github.event_name == 'pull_request'}}
+
+jobs:
+  Build:
+    runs-on: ubuntu-latest
```

Comment:
It could be nice to build an arm64 image as well, so the container works on Mac seamlessly, too.